Retitle “Configuring kubelet Garbage Collection"

Kartik494 opened the pull request:
This PR fixes #25300

kbhawkey commented on content/en/docs/concepts/cluster-administration/garbage-collection-container-images.md:
```diff
@@ -1,13 +1,13 @@
 <!-- overview -->
 
-Garbage collection is a helpful function of kubelet that will clean up unused images and unused containers. Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
+Garbage collection is a helpful function of kubelet that will clean up unused [images](https://kubernetes.io/docs/concepts/containers/#container-images) and unused [containers](https://kubernetes.io/docs/concepts/containers/). Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
```
Hi @Kartik494 .
Do you want to link to the containers page from the GC page or the other way around?
Also, I suggest to write the like:
```
[containers](/docs/concepts/containers/)
```

If you change the name of the file (not sure if required), you will want to add a redirect for the original page?
See `website/static/_redirects`.

Kartik494 replied:
Hi @kbhawkey i agreed i will be changing the title only and change the name of file to the original one  and i will be updated the link as you mentioned .
Thanks

Kartik494 commented:
Hi @kbhawkey  while i addressing the comments i observe that it took the domain name for github rather than k8s.io `(https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/cluster-administration/docs/concepts/containers/#container-images)` , so should i keep the link as  `(https://kubernetes.io/docs/concepts/containers/#container-images)`

kbhawkey commented on content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md:
```diff
@@ -1,13 +1,13 @@
 <!-- overview -->
 
-Garbage collection is a helpful function of kubelet that will clean up unused images and unused containers. Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
+Garbage collection is a helpful function of kubelet that will clean up unused [images](docs/concepts/containers/#container-images) and unused [containers](docs/concepts/containers/). Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
```
Hi @Kartik494 .
Would you add a forward slash before the link path:
`/docs/concepts/containers/`

Kartik494 replied:
Hi @kbhawkey i have addressed the comment as you suggested , but its again give the link address `https://github.com/kubernetes/website/blob/master/docs/concepts/containers/#container-images` not expected as k8s.io domain , could you please suggest!

kbhawkey commented:
Page preview:
https://deploy-preview-25457--kubernetes-io-master-staging.netlify.app/docs/concepts/cluster-administration/kubelet-garbage-collection/

kbhawkey commented:
Hi @Kartik494 . Looks great! ❇️ 
The links from the GC cluster-administration page to the container page seem fine.
@sftim, the issue suggested:

```
Hyperlink from within https://kubernetes.io/docs/concepts/containers/ to the page https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/
```
Instead, do you want to add a link **to** the GC page from this page:
`https://deploy-preview-25457--kubernetes-io-master-staging.netlify.app/docs/concepts/containers/#what-s-next`

kbhawkey commented:
/lgtm

tengqm commented:
/approve
